js1-week2-homework/assets/hyf_week2_q8.js

```diff
@@ -0,0 +1,29 @@
+let itemOne = 8;
+let itemTwo = 'Deh Haag';
+let itemThree = [1, 2, 3];
+let itemFour = { somthing: 1 };
```

Comment:
Again, any reason why you picked `let` instead of `const` here? 🙂

Reply:
I've got your message.